DataFormats/EgammaCandidates/src/Photon.cc

```diff
@@ -150,6 +151,7 @@ void Photon::setP4(P4type type, const LorentzVector& p4, float error, bool setTo
     case regression1:
       eCorrections_.regression1P4 = p4;
       eCorrections_.regression1EnergyError = error;
+      [[fallthrough]];
```

Comment:
Shouldn't it rather break here? @Sam-Harper

Reply:
yes, see above comment, thanks for spotting!